lib/benchpark/cmd/experiment.py

```diff
@@ -57,7 +57,9 @@ def setup_parser(root_parser):
     init_parser.add_argument(
         "--basedir", help="Generate a system dir under this, and place all files there"
     )
-    init_parser.add_argument("--system", help="The system this will be run on")
+    init_parser.add_argument(
+        "--system", help="The system this will be run on", required=True
+    )
```

Comment:
Maybe we don't do this yet, but this would be a simple way to enforce the use of system-aware experiment init

Reply:
I removed the change, just wanted to show it